components/script/dom/htmlcanvaselement.rs

```diff
@@ -159,18 +161,15 @@ impl HTMLCanvasElement {
     }
 
     fn recreate_contexts(&self) {
```

Comment:
Please rename this to `recreate_contexts_after_resize`. Now it's a bit weird reading only this method because it's just called `recreate`, but it calls `resize`.